test/src/page.spec.ts

```diff
@@ -111,6 +111,28 @@ describe('Page', function () {
         expect(message).not.toContain('Timeout');
       }
     });
+    it('closing last tab should not close the browser', async () => {
```

Comment:
Let's put this test the `browser.spec.ts` as it's not testing anything on the Page itself.

Reply:
done